parse/parser.go

```diff
@@ -191,17 +192,53 @@ func parseIdentifier(buf TokenBuffer) (ast.Expression, []error) {
 	return &ast.Identifier{Value: token.Val}, nil
 }
 
-// TODO: implement me w/ test cases :-)
 func parseIntegerLiteral(buf TokenBuffer) (ast.Expression, []error) {
-	return nil, nil
+	token := buf.Peek(CURRENT)
+	errs := make([]error, 0)
+
+	if token.Type != Int {
+		errs = append(errs, errors.New("parseIntegerLiteral() error - "+token.Val+" is not integer"))
+		return nil, errs
+	}
+
+	value, err := strconv.ParseInt(token.Val, 0, 64)
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
+	}
+	lit := &ast.IntegerLiteral{Value: value}
+	buf.Read()
+	return lit, nil
 }
 
-// TODO: implement me w/ test cases :-)
 func parseBooleanLiteral(buf TokenBuffer) (ast.Expression, []error) {
-	return nil, nil
+	token := buf.Peek(CURRENT)
+	errs := make([]error, 0)
```

Comment:
Instead of peek like this statement `token := buf.Peek(CURRENT)`,
It would be better to just read token`token := buf.Read()`!!